### ISSUE TITLE
test_ARAX_workflows.py test_example_2 failing in master?

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1385]
I'm seeing this test failing in `master` in my local dev environment:
```
pytest -v -s test_ARAX_workflows.py -k test_example_2 --runslow
```
```
FAILED test_ARAX_workflows.py::test_example_2 - assert 6 == 15
```
looks like it's an Overlay DTD-related test. I believe I have the latest DTD databases and everything; I saw them auto-download recently (after which this test started failing, I think?). maybe just an issue with the number of answers the test expects?

**Comment by dkoslicki** [comment]
@finnagin and @chunyuma any input on this one?

**Comment by chunyuma** [comment]
Hi @amykglen, based on this DSL query, it seems like the `filter_kg` or `filter_results` module doesn't do any filtering for DTD -related edges. So I'm wondering if this is associated with `overlay(action=compute_jaccard` because it does filter some `jaccard_index` edges.

**Comment by chunyuma** [comment]
Ah, I find the reason. It is associated with one modification that I did for #1372 last night. Currently, the `overlay(action=predict_drug_treats_disease)` allows to set `threshold` and `slow_mode` like what we did in `expand(kg=DTD)`. So the default threshold is 0.8 and using the DTD database rather than the DTD model to get the DTD probability. 

If we set `overlay(action=predict_drug_treats_disease, subject_qnode_key=n02, object_qnode_key=n00, virtual_relation_label=P1, threshold=0)` in this DSL query, it fixes this issue. I've made a push and updated this DSL. Now this test passes.
